### PR TITLE
Fixes to details page

### PR DESCRIPTION
**Pull request by gemfarmer**
[CircleCI](https://circleci.com/gh/18F/nsf-sbir/tree/address)

[:sunglasses: PREVIEW successful details page](http://localhost:4000/site/portfolio/details/?company=the-echo-nest-corporation#the-echo-nest-corporation)
[🙃  PREVIEW unsuccessful details page](https://federalist.fr.cloud.gov/preview/18f/nsf-sbir/address/portfolio/details/?company=fake-company-name)

Changes proposed in this pull request:
- Adds an `<address>` tag to the details page
- Adds a catch for a query string that isn't valid within the details page
- Changes the program directors compiler so that @femmebot's new content isn't removed on build

/cc @line47

### CODE REVIEW

File: _layouts/details.html [hunk 1]
```diff
@@ -8,6 +8,11 @@
     <h2 class="h3 m0">Loading company results...</h2>
   </div>
 
+  <div class="results-failure flex flex-column usa-grid usa-section" style="display: none;">
+    <h2 class="h3 m0">There were no results found for <span class="results-query text-xl background-secondary font-mono">that query</span>.</h2>
+    <p>Return to <span class="results-previous-page fake-link" tabindex="0">previous page</span> to try again.</p>
```
Comment (line47) [inline _layouts/details.html]: Should we wire up this link?

**Comment by line47** [comment]
I also noticed some weird " . " after the abstract text.  See screenshot
screen shot 2017-06-26 at 3 19 06 pm

**Comment by gemfarmer** [comment]
@line47 good catches! I made some changes. Let me know what you think..

**Comment by line47** [comment]
@gemfarmer Yes yes, nice!

**Pull request merged**